sdk/core/azure-core/src/main/java/com/azure/core/models/CloudEvent.java

```diff
@@ -305,6 +307,7 @@ public CloudEvent(String source, String type, BinaryData data, CloudEventDataFor
         this.id = CoreUtils.randomUuid().toString();
         this.specVersion = CloudEvent.SPEC_VERSION;
         this.binaryData = data;
+        this.time = Instant.now().atOffset(ZoneOffset.UTC);
```

Comment:
nit: [Instant.now](https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html#now--) is already UTC

Reply:
Was told `this.time` is an `OffsetDateTime`, so `OffsetDateTime.now(ZoneOffset.UTC)` instead